### PR TITLE
have workflow doc represent reality

### PR DESCRIPTION
**Pull request by afeld**
Closes https://github.com/18F/tts-tech-portfolio/issues/907.

This pull request is meant to update the workflow doc to reflect the ceremonies, board, cards, labels, etc. as we've been doing/using them. Nothing in there should be a surprise. Notable changes:

- Removed themes and epics
- Defined what "groomed" means
- Added in the newer ceremonies (coworking and icebox thaw)
- Added more guidance around prioritization
- Various formatting improvements

Reviewing the overall diff won't be that useful. I suggest you either review commit by commit, or just [read the rewritten doc](https://github.com/18F/tts-tech-portfolio/blob/workflow-update/how_we_work/workflow.md) and give feedback on that. Anything that's unclear? Anything that's missing?

**Hoping to merge before 6/1 when @adborden starts.**

### CODE REVIEW

File: how_we_work/workflow.md [hunk 1]
```diff
@@ -1,185 +1,86 @@
...
+1. **Review `Icebox` cards:** Move cards from `Icebox` into `Backlog (Sprint Staging)` that you would like to discuss adding into the Sprint.
+1. **Get `Ready`:** Move cards between `Backlog (Sprint Staging)` and `Ready (Sprint Planned)`.
+   - `Ready (Sprint Planned)` should represent the [highest-priority](#prioritization) issues that haven't been started.
+   - There should only be as many assigned to each person as they think they will be able to start this Sprint. Keep the following in mind:
+     - [What to work on](#what-to-work-on)
+     - Upcoming leave
+     - Other commitments (training, working groups, etc.)
 
-**Purpose:** Team bonding/culture-building.
+#### Prioritization
```
Comment (JJediny) [inline how_we_work/workflow.md:25]: I like the new section!

**Review by JJediny — APPROVED** [review]
Approved, but suggest we move `grooming status` as a sub heading under `grooming`. Also not much defining/stressing the strict use of `Implementation Steps` vs `Acceptance Criteria` happy for those to be unspoken and felt out during grooming sessions though too. So LGTM for now

**Comment by afeld** [comment]
Will see if I can clarify in a subsequent pass. Thanks!

**Comment by afeld** [comment]
> suggest we move `grooming status` as a sub heading under `grooming`

I intentionally moved it under `Labels` to distinguish grooming as a status versus grooming as a ceremony.

Tried to make both more clear under https://github.com/18F/tts-tech-portfolio/pull/1354.

**Pull request merged**